automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterExceptionSpec.scala

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
+import org.broadinstitute.dsde.workbench.service.RestException
+import org.scalatest.{FreeSpec, ParallelTestExecution}
+
+class ClusterExceptionSpec extends FreeSpec with LeonardoTestUtils with ParallelTestExecution with BillingFixtures {
```

Comment:
tests for expected exception thrown by leo

Reply:
Ok - previously I had bundled up these tests in `ClusterConcurrencySpec` because they are verifying errors thrown by concurrent operations to clusters. But I'm fine with moving them here.